modules/rubiconBidAdapter.js

```diff
@@ -757,14 +766,18 @@ export const spec = {
             advertiserId: ad.advertiser, networkId: ad.network
           },
           meta: {
-            advertiserId: ad.advertiser, networkId: ad.network
+            advertiserId: ad.advertiser, networkId: ad.network, mediaType: 'banner'
```

Comment:
Nitpick for consistency, does it make sense to use the imported const, BANNER, instead of a string, as you are for VIDEO meta.mediaType?

Reply:
Yep good idea. Updated